components/icons/icon.js

```diff
@@ -0,0 +1,145 @@
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { loadSvg } from '../../.generated/icons/presetIconLoader.js';
+import { RtlMixin } from '../localize/rtl-mixin.js';
```

Comment:
You'll want to get my changes from master and update path here. `../../mixins/rtl-mixin.js`

Reply:
Done